app/component/app-main/index.js

```diff
@@ -11,10 +11,10 @@ angular.module('demoApp').directive('appMain', function(){
     controller: [MainController],
     controllerAs: 'mainCtrl',
     bindToController: true,
-    scope: {},
```

Comment:
Since this file is no longer being used it should not be included in the code base :)